example/scatter_plot.html

```diff
@@ -44,8 +44,8 @@
         var trend = new Nugget.TrendLine({
             dataSeries: dataSeries,
             color: 'yellow',
-            slope: 5000000,
-            offset: 50000000
+            slope: Infinity,
+            origin: { x: 0, y: 0 }
```

Comment:
Changed the signature a little so it's easier for the caller to use - `origin` is much easier to explain than `offset` to non-math geeks :)